ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/system-control-plane.j2

```diff
@@ -0,0 +1,38 @@
+{# eos - system #}
+{% if system.control_plane is arista.avd.defined %}
+!
+system control-plane
+{# control_plane tcp mss ceiling #}
+{%    if system.control_plane.tcp_mss.ipv4 is arista.avd.defined and system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}
+{%       set cp_mss_cli = "tcp mss ceiling ipv4 " ~ system.control_plane.tcp_mss.ipv4 ~ " ipv6 " ~ system.control_plane.tcp_mss.ipv6 %}
+{%    elif system.control_plane.tcp_mss.ipv4 is arista.avd.defined and system.control_plane.tcp_mss.ipv6 is not arista.avd.defined %}
+{%       set cp_mss_cli = "tcp mss ceiling ipv4 " ~ system.control_plane.tcp_mss.ipv4 %}
+{%    elif system.control_plane.tcp_mss.ipv4 is not arista.avd.defined and system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}
+{%       set cp_mss_cli = "tcp mss ceiling ipv6 " ~ system.control_plane.tcp_mss.ipv6 %}
+{%    endif %}
+  {{ cp_mss_cli }}
+{# control_plane access_group ipv4 #}
+{%    if system.control_plane.ipv4_access_group is arista.avd.defined %}
```

Comment:
The `arista.avd.natural_sort` filter includes this check, so this line can be removed. Same in other for loops.

Reply:
Removed the line since `arista.avd.natural_sort` takes care of this, thanks!